frontend/src/app/google-charts/google-charts.component.ts

```diff
@@ -69,8 +95,58 @@ export class GoogleChartsComponent implements AfterViewInit {
     });
   }
 
+  getEventAnnotation(currentDate, countryEvents, eventIndex): any[] {
```

Comment:
Could you please add some comments for some of these functions? This file is getting bigger and e.g. here it would be convenient to see how the input and output look like without going through the code.

Reply:
done